scripts/get-latest-rhocp-repo.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Following script queries RHOCP repositories to get latest available
+# for version of MicroShift from current branch.
+# It expect system to be registered (i.e. entitlement exists).
+#
+# We cannot use branch version (or sometimes even previous minor one)
+# because repositories are only usable after the release.
+# Accessing them before the release results in 403 error.
+#
+# Output is just the minor (Y) version number.
+
+set -o pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPOROOT="$(cd "${SCRIPTDIR}/.." && pwd)"
+
+if ! sudo subscription-manager status >&/dev/null; then
+    >&2 echo "System must be subscribed"
+    exit 1
+fi
+
+cacert="/etc/rhsm/ca/redhat-uep.pem"
+cert=$(find /etc/pki/entitlement -iname '*.pem' -not -iname '*-key.pem')
+key=$(find /etc/pki/entitlement -iname '*-key.pem')
+
+current_minor=$(make -C "${REPOROOT}" debug | grep "MINOR:" | cut -d':' -f2)
+stop=$(( current_minor - 3 ))
+
+for ver in $(seq "${current_minor}" -1 "${stop}"); do
```

Comment:
Can we add some comments here explaining what's going on?

Reply:
added comment